service/azureconfig/v1/resource/deployment/deployment.go

```diff
@@ -10,6 +10,11 @@ import (
 const (
 	mainDeploymentName     = "cluster-main-template"
 	templateContentVersion = "1.0.0.0"
+
```

Comment:
Pulling these all the way from the CR doesn't really make sense to me, it's not like they're configurable.